apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -165,3 +166,39 @@ function showHomepage() {
     document.getElementById('homepage-container')
   );
 }
+
+window.CleverTakeoverManager = function (options) {
+  this.options = options;
+  const self = this;
+
+  const linkCleverDiv = $('<div>');
+  function showLinkCleverModal(cancel, submit) {
+    $(document.body).append(linkCleverDiv);
+
+    ReactDOM.render(
+      <LinkCleverAccountModal
+        isOpen={true}
+        handleCancel={cancel}
+        handleSubmit={submit}
+      />,
+      linkCleverDiv[0]
+    );
+  }
+
+  if (self.options.cleverLinkFlag) {
+    showLinkCleverModal(onCancelModal, onConfirmLink);
+  }
+
+  function closeLinkCleverModal() {
```

Comment:
Nit: This one-line function is only used in one place - I'd just move the body of this function to `onCancelModal`.